games/tests/integration/testcases.py

```diff
@@ -56,35 +56,45 @@ def setUp(self):
         self.player_client.force_login(self.player.user)
         self.opponent_client.force_login(self.opponent.user)
 
+        self.player.client = self.player_client
```

Comment:
Most of the significant changes in this PR are in this file. Basically everything else is updates to use the new hotness.